cunumeric/random/random.py

```diff
@@ -109,8 +109,12 @@ def randint(
     Multiple GPUs, Multiple CPUs
     """
 
+    if not isinstance(low, int) or not (high is None or isinstance(high, int)):
+        raise NotImplementedError("'low' and 'high' must be integers")
```

Comment:
`high` cab be None, so splitting this into two checks would afford more fine-grained error messages. But if this is copied from Numpy then probably not worth changing.